code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1253,21 +1253,24 @@ datum
 
 			on_mob_life(var/mob/M, var/mult = 1)
 				if (!M) M = holder.my_atom
-				M.take_toxin_damage(1*mult)
-				random_brute_damage(M, 1*mult, FALSE)
+				M.take_toxin_damage(0.5*mult)
+				take_bleeding_damage(M, null, 2 * mult, DAMAGE_CUT, 1, get_turf(M))
```

Comment:
```suggestion
				take_bleeding_damage(M, null, 2 * mult, DAMAGE_CUT)
```
Nit but you don't need these two additional arguments as the proc already sets them *and* they don't provide extra clarity.